app/src/main/java/com/example/fragmentassg2/fragments/FragmentLogin.kt

```diff
@@ -58,11 +68,14 @@ class FragmentLogin : Fragment(), View.OnClickListener {
     private fun login(){
         val number ="9953755955"
```

Comment:
@raghav32 Create a constant file in Util folder ( Create a package named "Util") and place all the constant files over there